healthcheck/core/src/main/java/org/apache/felix/hc/core/impl/executor/async/cron/CronSchedulerTask.java

```diff
@@ -0,0 +1,53 @@
+package org.apache.felix.hc.core.impl.executor.async.cron;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public final class CronSchedulerTask implements Runnable {
+
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+
+    public final CronJob cronJob;
+    public final CronScheduler scheduler;
+    public final CronParser sequenceGenerator;
+
+    volatile boolean executing;
+    volatile long lastExecutingTime = 0;
+    volatile long nextExecutingTime = 0;
+
+    public CronSchedulerTask(final CronScheduler scheduler, final CronJob cronJob,
+            final CronParser sequenceGenerator) {
+        this.scheduler = scheduler;
+        this.cronJob = cronJob;
+        this.sequenceGenerator = sequenceGenerator;
+    }
+
+    @Override
+    public void run() {
+        try {
+        	cronJob.run();
```

Comment:
Indent should be 2 spaces to left

Reply:
In Eclipse, this looks good but due to formatting standard in GitHub, it looks different here.